### ISSUE TITLE
Allow searching for classes

### ISSUE THREAD

**Issue by nicolas-raoul** [joseignm/GraFa#5]
I wanted to search for types of cheese (camembert, bleu, etc).

Obviously [camembert](https://www.wikidata.org/wiki/Q131480) is a subclass of cheese, not an instance of cheese.

I haven't found a way to search for this.

**Comment by aidhog** [comment]
Yes, at the moment we require starting with a keyword search or a type, while it seems classes/types in Wikidata don't have an instance-of value. Probably the best way to address this would be to build a super-facet over all entities and just let users skip the type selection. This is something we will postpone for now, however, since computing that super-facet is not trivial, and the interface would require adjusting maybe to not show all facet-properties at once.
